.compose/ssl/Dockerfile

```
@@ -0,0 +1,23 @@
+FROM alpine:3.21
+
+ARG MKCERT_VERSION="v1.4.4"
+ARG TARGETARCH
+ARG DOMAINS="docker.internal *.docker.internal localhost"
+
+RUN apk add --no-cache openssl curl && \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)